engine/env/spark/src/main/java/org/datacleaner/spark/SparkJobContext.java

```diff
@@ -220,11 +232,49 @@ public String getResultPath() {
         }
         return null;
     }
-    
+
+    /**
+     * Gets the job name (removing the extension '.analysis.xml')
+     * 
+     * @return
+     */
     public String getAnalysisJobName() {
         final int lastIndexOfSlash = _analysisJobXmlPath.lastIndexOf("/");
         final int lastIndexOfFileExtension = _analysisJobXmlPath.lastIndexOf(".analysis.xml");
-        final String jobName = _analysisJobXmlPath.substring(lastIndexOfSlash+1, lastIndexOfFileExtension);
-        return  jobName;
+        final String jobName = _analysisJobXmlPath.substring(lastIndexOfSlash + 1, lastIndexOfFileExtension);
+        return jobName;
+    }
+
+    /**
+     * Gets the hdfs path of job's result
+     * 
+     * @return
+     */
+    public String getResultJobFilePath() {
+        String resultPath = getResultPathUsercustomized();
+        final Configuration hadoopConfiguration = _sparkContext.hadoopConfiguration();
+        final String fileSystemPrefix = hadoopConfiguration.get("fs.defaultFS");
+        if (resultPath == null || resultPath.isEmpty()) {
+            resultPath = fileSystemPrefix + DEFAULT_RESULT_PATH;
+        } else {
+            final URI uri = URI.create(resultPath);
+            if (!uri.isAbsolute()) {
+                if (!resultPath.startsWith("/")) {
+                    resultPath = "/" + resultPath;
```

Comment:
doesn't the path normally start with "hdfs://.."? In that case we are now constructing a path like "/hdfs://.." I think